examples/specs/layer_ranged_dot.vl.json

```diff
@@ -0,0 +1,85 @@
+{
+  "$schema": "https://vega.github.io/schema/vega-lite/v2.json",
+  "description": "A ranged dot plot that uses 'layer' to convey changing life expectancy for the five most populous countries (between 1955 and 2000).",
+  "data": {"url": "data/countries.json"},
+  "transform": [
+      {
+          "filter": {
+              "field": "country",
+              "oneOf": ["China", "India"
+              , "United States", "Indonesia", "Brazil"]
+          }
+      },
+      {
+          "filter": {
+              "field": "year",
+              "oneOf": [1955, 2000]
+          }
+      }
+  ],
+  "layer": [
+      {
+          "mark": "point",
+          "encoding": {
+              "x": {
+                  "field": "life_expect",
+                  "type": "quantitative",
+                  "axis": {"title": "Life Expectancy (years)"}
+              },
+              "y": {
+                  "field": "country",
+                  "type": "nominal",
+                  "axis": {
+                      "title": "Country",
+                      "offset": 5,
+                      "ticks": false,
+                      "minExtent": 70,
+                      "domain": false
+                  }
+              },
+              "color": {
+                  "field": "year",
+                  "type": "ordinal",
+                  "scale": {
+                      "domain": [1955, 2000],
+                      "range": ["#e6959c",
+                      "#911a24"]
+                  },
+                  "legend": {
+                      "title": "Year"
+                  }
+              }
+          }
+      },
+      {
+          "mark": "line",
+          "encoding": {
+              "x": {
+                  "field": "life_expect",
+                  "type": "quantitative"
+              },
+              "y": {
+                  "field": "country",
+                  "type": "nominal"
+              },
+              "opacity": {
+                  "field": "country",
+                  "type": "nominal",
+                  "scale": {
+                      "range": [0.2, 0.2]
+                  },
+                  "legend": null
+              }
+          }
+      }
+  ],
+  "config": {
+      "point": {
```

Comment:
Since there is only one layer of point and line, perhaps using
`mark:{ type: 'point', filled: true}`
and `size: {value: 100}` and `color: {"value": "#db646f"}` are better.